lib/index.js

```diff
@@ -34,16 +51,53 @@ export const decorateHeader = (Header, { React, PropTypes }) => {
       return {
         theme: PropTypes.object,
         network: PropTypes.string,
+        setCurrentClient: PropTypes.func.isRequired,
+        clients: PropTypes.shape({
+          clients: PropTypes.objectOf(
+            (obj, key, componentName, location, propFullName) => {
+              if (!obj[key].chain)
+                return new Error(
+                  'Invalid prop `' +
```

Comment:
Did the autoformatter do this? Looks strange

Reply:
Yeah. This was prettier and I agree it looks a little funny. This is how it deals with long template literal strings.